pymlx/featurizer.py

```diff
@@ -7,13 +7,40 @@
 from .features_handler import *
 
 
+def listable(t):
+    return Union[t, List[t]]
+
+
 class Featurizer:
-    def __init__(self, handlers: List[FeaturesHandler], sparse=False):
-        assert isinstance(handlers, list)
+    def __init__(self,
+                 handlers: listable(FeaturesHandler) = None,
```

Comment:
I would put `handlers` at last and call it `custom_handlers`

Reply:
Updated